Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM alpine:3.18.4
+
+ARG USER=ext-installer
+ENV HOME /home/$USER
+
+RUN apk update && apk add file curl
```

Comment:
Does this command root access? you can downgrade the access later after installing.

Reply:
@ashutosh16 that is what I am doing at line 9

Reply:
Got it. thx